### PR TITLE
Prefix defaults.yaml components with -v2

### PR DESCRIPTION
**Pull request by raminqaf**

### CODE REVIEW

File: defaults.yaml [hunk 1]
```diff
@@ -1,31 +1,29 @@
-kubernetes-app:
+helm-app:
```
Comment (disrupted) [inline defaults.yaml:1]: why did you change this?

Comment (raminqaf) [reply defaults.yaml:1]: IMO streams-bootstrap is a Helm-App. We use Helm charts and Helm CLI to install/unistall it

Comment (disrupted) [reply defaults.yaml:1]: but namespace applies to all kubernetes-apps
imo it's not necessary to change it

Comment (raminqaf) [reply defaults.yaml]: Okay reverted

**Review by disrupted — APPROVED** [review]
